orion-server/src/main/java/com/pinterest/orion/core/actions/aws/ReplaceEC2InstanceAction.java

```diff
@@ -152,6 +152,13 @@ public void runAction() {
           );
           getEngine().alert(AlertLevel.MEDIUM, msg);
           getEngine().alert(AlertLevel.HIGH, msg);
+          OrionServer.metricsCounterInc(
+                  "ec2ReplacementGetState",
```

Comment:
Let's clarify whether this is for old or new instance.

Reply:
OK. Will add more context